ddr/tools/ddrgen/Makefile

```diff
@@ -84,3 +78,12 @@ OBJECTS = \
   ddrgen$(OBJEXT)
 
 include $(top_srcdir)/omrmakefiles/rules.mk
+
+# This must follow the include above because we want to append to LD_STATIC_LIBS.
+ifeq (zos,$(OMR_HOST_OS))
+  ifeq (1,$(OMR_ENV_DATA64))
+    LD_STATIC_LIBS += /usr/lpp/cbclib/lib/libelfdwarf64.x
+  else
+    LD_STATIC_LIBS += /usr/lpp/cbclib/lib/libelfdwarf32.x
```

Comment:
I think we'll need a similar change in CMake somewhere. Not sure what the best place is though.

Reply:
I expect that's true, but it seems CMake never fully supported ddr because the extra required libraries aren't specified for any platform. I think it's safe to defer those additions.

Reply:
I'm fine with that. @dnakamura FYI we'll need these changes eventually once DDR support comes closer.